ios/GLCanvas.m

```diff
@@ -320,7 +321,9 @@ - (void)drawRect:(CGRect)rect
     });
   }
   else {
+    RCTProfileBeginEvent(0, @"GLCanvas render", nil);
     [self render];
+    RCTProfileEndEvent(0, @"gl", nil);
```

Comment:
why is there 2 different messages? "GLCanvas render" and "gl" shouldn't be the same? (no idea how this works)